dev-python/pygame/python_pygame-1.9.4.recipe

```diff
@@ -8,13 +8,13 @@ COPYRIGHT="2000-2004, 2007 Pete Shinners
 	2007 Richard Goedeken
 	2007-2008 Marcus von Appen"
 LICENSE="GNU LGPL v2.1"
-REVISION="1"
-SOURCE_URI="https://files.pythonhosted.org/packages/b2/6b/c510f0853765eb2219ca5aa3d416d65bb0dea7cd9bb2984aea0a0e04c24d/pygame-1.9.4.tar.gz"
+REVISION="2"
+SOURCE_URI="https://files.pythonhosted.org/packages/b2/6b/c510f0853765eb2219ca5aa3d416d65bb0dea7cd9bb2984aea0a0e04c24d/pygame-$portVersion.tar.gz"
```

Comment:
```suggestion
SOURCE_URI="https://files.pythonhosted.org/packages/source/p/pygame/pygame-$portVersion.tar.gz"
```

Reply:
Saw this, need to adjust the checksum for it also then

Reply:
Updated, no need to update the checksum :) Thanks!